src/notebooks/gen_yolo_training_data.ipynb

```diff
@@ -0,0 +1,234 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# An example notebook how to generate training data for YOLOv5 from an .EMObs file"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "outputs": [],
+   "source": [
+    "import os, sys\n",
+    "sys.path.append(\"..\")\n",
+    "\n",
+    "import emtmlibpy.emtmlibpy as emtm\n",
+    "from emtmlibpy.emtmlibpy import EMTMResult\n",
+    "\n",
+    "import pandas as pd\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "outputs": [],
+   "source": [
+    "SRC_DIR = os.path.abspath(os.getcwd())\n",
+    "TEST_FILES_PATH = os.path.join(SRC_DIR, '..', 'test_files')"
```

Comment:
This is obviously fine, but I just thought I'd mention [`pathlib`](https://docs.python.org/3/library/pathlib.html#id5), which is a very nice standard library replacement for all of the old `os.path` helpers. E.g. you could use here:

```suggestion
    "import os, sys\n",
    "from pathlib import Path\n",
    "sys.path.append(\"..\")\n",
    "\n",
    "import emtmlibpy.emtmlibpy as emtm\n",
    "from emtmlibpy.emtmlibpy import EMTMResult\n",
    "\n",
    "import pandas as pd\n"
   ],
   "metadata": {
    "collapsed": false,
    "pycharm": {
     "name": "#%%\n"
    }
   }
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "outputs": [],
   "source": [
    "SRC_DIR = Path.cwd().resolve()\n",
    "TEST_FILES_PATH = SRC_DIR / '..' / 'test_files'"
```
With a small caveat that the above would also resolve symlinks, while `os.abspath()` would not. Presumably that would be of no concern, of course.